Sources/DCoreKit/Net/Model/Request/BaseRequest.swift

```diff
@@ -57,7 +57,7 @@ extension BaseRequest: Encodable {
         // json rpc group and api
         var unkeyed = keyed.nestedUnkeyedContainer(forKey: .params)
         
-        try unkeyed.encode(group.id)
+        try unkeyed.encode(group.name)
```

Comment:
I would remove `id` property completely. Do we need it?

Reply:
Removed